addons/binding/org.openhab.binding.autelis/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,568 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="autelis"
```

Comment:
Please check if any of the newer features for thing desciptions can be applied here, like "advanced" channels (so that only the most important is shown by default), channel categories (for having suitable icons) and possibly channel groups. See docs at https://github.com/eclipse/smarthome/blob/master/docs/sources/architecture/thing-definition.md#state-description